mqtt-client-Go/on_way-tls/main.go

```diff
@@ -39,37 +40,50 @@ func main() {
 		Cafile:   *cafile,
 	}
 	client := mqttConnect(&config)
-	go sub(client, &config)
+	go subscribe(client, &config)
```

Comment:
You can add the subscription to the OnConnect callback.

Reply:
done